### PR TITLE
handle new sort-key when sorting tables

### PR DESCRIPTION
**Pull request by aschnell**
Handle the new sort-key when sorting tables.

See libyui/libyui#156.

### CODE REVIEW

**Comment by aschnell** [comment]
The Travis failure is due to some OBS being down? Anyway, even if it would not be down the docker image would likely be outdated (see mail on yast-devel).

File: package/libyui-qt-doc.spec [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 Name:           %{parent}-doc
 # DO NOT manually bump the version here; instead, use   rake version:bump
-Version:        2.52.0
+Version:        2.52.1
```
Comment (jreidinger) [inline package/libyui-qt-doc.spec]: to be honest I am a bit lost about that API/ABI incompatible changes and versioning. So lets hope that increase of patch level is enough.

File: src/QY2ListView.cc [hunk 2]
```diff
@@ -418,29 +412,55 @@ QY2ListViewItem::operator< ( const QTreeWidgetItem & otherListViewItem ) const
...
         return retval;     // int < int
     else if (ok1 && !ok2)
         return true;       // int < string
     else if (!ok1 && ok2)
         return false;      // string > int
 
-    // and finally non-numeric sorting is done by the base class
-    return QTreeWidgetItem::operator<(otherListViewItem);
+    // and finally non-numeric sorting is done locale aware
+    return QString::localeAwareCompare(text1, text2) < 0;   // string < string
```
Comment (jreidinger) [inline src/QY2ListView.cc]: just curious is ncurses sorting same regarding locales? ( I mean also locale aware and if it is use same algorithm in background? )

Comment (aschnell) [reply src/QY2ListView.cc]: ncurses use wcscoll which is also locale aware. What Qt does internally I do not know.

File: package/libyui-qt-doc.spec (hunk 1 above)
Comment (aschnell) [reply package/libyui-qt-doc.spec]: That is indeed a problem with libyui.

I would have increased the minor version of the library but that does not work in the libyui setup (see comment in VERSION.cmake).

**Pull request merged**